src/main/java/oley/commands/Oley.java

```diff
@@ -0,0 +1,254 @@
+        String file = "./data/Oley.txt";
+        if (!tasks.isEmpty()) {
+            try {
+                changeFile(file);
+            } catch (IOException e) {
+                System.out.println("    OOPS, we have encountered an error!");
+                System.out.println("    Write to file failed.");
+            }
+        } else {
+            try {
+                clearFile(file);
+            } catch (IOException e) {
+                System.out.println("    OOPS, we have encountered an error!");
+                System.out.println("    Clear file failed.");
+            }
+        }
+    }
+
+    public static void exit() {
+        System.out.println("    Bye~ Feel free to talk to me anytime. I will always be here waiting for you. ฅʕ•̫͡•");
+    }
+
+    public static void lineBreaker() {
+        System.out.println(" ");
+        System.out.println("    🌙~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~🌙");
+    }
+
+    public static void printTask() {
+        System.out.println("    Here are the tasks in your list:");
+        int i = 0;
+        while (i < tasks.size()) {
+            System.out.println("    " + (i + 1) + "." + tasks.get(i).toString());
+            i++;
+        }
+    }
+
+    public static void mark(String sentence, boolean isBegin) {
+        String[] markInstructions = sentence.split(" ");
+        int toBeMarked = Integer.parseInt(markInstructions[1]) - 1;
+        if (toBeMarked >= tasks.size()) {
+            System.out.println("    You have not created Task " + (toBeMarked + 1) + " yet. Jiayous. I will always support you. ฅ •ﻌ•♡");
+        } else if (markInstructions[0].equals("mark")) {
+            if (tasks.get(toBeMarked).checkDone()) {
+                System.out.println("    This task has been marked as done already!");
+            } else {
+                tasks.get(toBeMarked).setDone();
+                if (!isBegin) {
+                    System.out.println("    Good job! I've marked this task as done:");
+                    System.out.println("    " + tasks.get(toBeMarked));
+                }
+            }
+        } else if (markInstructions[0].equals("unmark")) {
+            if (!tasks.get(toBeMarked).checkDone()) {
+                System.out.println("    This task hasn't been done yet!");
+            } else {
+                tasks.get(toBeMarked).setNotDone();
+                if (!isBegin) {
+                    System.out.println("    Sure~ I've marked this task as not done yet:");
+                    System.out.println("    " + tasks.get(toBeMarked));
+                }
+            }
+        }
+        if (!isBegin) {
+            String file = "./data/Oley.txt";
+            try {
+                changeFile(file);
+            } catch (IOException e) {
+                System.out.println("    OOPS, we have encountered an error!");
+                System.out.println("    Write to file failed.");
+            }
+        }
+    }
+
+    public static void instructions() {
```

Comment:
Why is most of the logic still in the main file? Please follow the increment instructions and refactor the code according to A-MoreOOP at minimal.